cli/test/org/partiql/cli/CliTest.kt

```diff
@@ -47,125 +43,142 @@ class CliTest {
         Files.deleteIfExists(testFile.toPath())
     }
 
-    private fun makeCli(
-        query: String,
-        input: String? = null,
-        bindings: Bindings<ExprValue> = Bindings.empty(),
-        outputFormat: OutputFormat = OutputFormat.ION_TEXT,
-        output: OutputStream = this.output,
-        compilerPipeline: CompilerPipeline = CompilerPipeline.standard(ion)
-    ) =
-        Cli(
-            valueFactory,
-            input?.byteInputStream(Charsets.UTF_8) ?: EmptyInputStream(),
-            output,
-            outputFormat,
-            compilerPipeline,
-            bindings,
-            query
-        )
-
-    private fun Cli.runAndOutput(): String {
-        run()
-        return output.toString(Charsets.UTF_8.name())
-    }
-
-    private fun String.singleIonExprValue() = valueFactory.newFromIonValue(ion.singleValue(this))
-    private fun Map<String, String>.asBinding() =
-        Bindings.ofMap(this.mapValues { it.value.singleIonExprValue() })
-
-    private fun assertAsIon(expected: String, actual: String) =
-        assertEquals(ion.loader.load(expected), ion.loader.load(actual))
-
     @Test
     fun runQueryOnSingleValue() {
-        val subject = makeCli("SELECT * FROM input_data", "{a: 1}")
-        val actual = subject.runAndOutput()
+        val query = "SELECT * FROM input_data"
+        val input = "[{'a': 1}]"
+        val expected = "$partiqlBagAnnotation[{a: 1}]"
 
-        assertAsIon("{a: 1}", actual)
+        val ionInputResult = makeCliAndGetResult(query, input, inputFormat = InputFormat.ION)
+        val partiqlInputResult = makeCliAndGetResult(query, input, inputFormat = InputFormat.PARTIQL)
```

Comment:
Thanks for adding some tests with the `InputFormat.PARTIQL`. I think we may need a couple additional tests:
- use `InputFormat.PARTIQL` to load an input that uses PartiQL syntax (e.g. a bag with the angle bracket syntax: `<<{'a': 1}>>`)
- use `InputFormat.ION` to load an input with PartiQL syntax (e.g. bag w/ angle brackets), which should error

AFAICT, the inputs for these tests just use Ion syntax and nothing specific to PartiQL.